### ISSUE TITLE
Pretty print JSON logs

### ISSUE THREAD

**Issue by andrewbanchich** [ory/oathkeeper#876]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/oathkeeper/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/oathkeeper/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [X] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe your problem

Oathkeeper logs are pretty hard to parse with the naked eye.

### Describe your ideal solution

Add a `format: json-pretty` config option for pretty printing logs.

### Workarounds or alternatives

None

### Version

0.38.15-beta.1

**Comment by aeneasr** [comment]
👍
